vtex/faststore
# Feat: Adds `Filter` and `FilterSlider` Components.

## What's the purpose of this pull request?

- [x] Adds `Filter `, `FilterSlider` and its related components to `faststore/ui` and `faststore/components`.
- [x]  Changes `Filter` on `faststore/core` with `FilterFixed` (Desktop) and `FilterSlider` (Mobile) options.
- [x]  Adds `Filter` component docs page.
- [x]  Adds `FilterSlide` component docs page.

## How to test it?
- You can see the components page in our nextra doc from the preview [Filter](https://faststore-site-rgu9j17br-faststore.vercel.app/components/organisms/filter) and [FilterSlider](https://faststore-site-rgu9j17br-faststore.vercel.app/components/organisms/filter-slider).
- In the store you can run `yarn dev` and checks the `Filter` and `FilterSlider` (Mobile) component locally or test it using the preview link.


### Starter PR
https://github.com/vtex-sites/starter.store/pull/25
